indexer/ai/src/main/java/com/google/cloudsearch/ai/StandardSkillEntityExtraction.java

```diff
@@ -0,0 +1,200 @@
+    {
+        this.parse(aiSkill);
+    }
+
+    @Override
+    public void setAISkillName(String aiSkillName) {
+        this.aiSkillName = aiSkillName;
+    }
+
+    @Override
+    public String getAISkillName() {
+        return this.aiSkillName;
+    }
+
+    @Override
+    public void setOutputMappings(JSONArray outputMapping) {
+
+        Iterator<JSONObject> mappingIterator = outputMapping.iterator();
+        while(mappingIterator.hasNext()){
+            JSONObject mappingObject = mappingIterator.next();
+            OutputMapping obj = new OutputMapping();
+            obj.setPropertyNames((String) mappingObject.get(Constants.configTargetProperty));
+            obj.setSkillOutputField((String) mappingObject.get(Constants.configOutputField));
+            this.outputMappings.add(obj);
+        }
+    }
+
+    @Override
+    public List<OutputMapping> getOutputMappings() {
+        return this.outputMappings;
+    }
+
+    @Override
+    public void setInputs(JSONObject input) {
+        if(input.get("language") != null){
+            this.inputLanguage = (String) input.get("language");
+        }
+
+        if(input.get("encoding") != null){
+            switch((String) input.get("encoding")){
+                case "UTF8" : {
+                                this.inputEncoding = EncodingType.UTF8;
+                                break;
+                }
+                case "UTF16" : {
+                                this.inputEncoding = EncodingType.UTF16;
+                                break;
+                }
+                case "UTF32" : {
+                                this.inputEncoding = EncodingType.UTF32;
+                                break;
+                }
+            }
+        }
+    }
+
+    @Override
+    public JSONObject getInputs() {
+
+        JSONObject obj = new JSONObject();
+        obj.put("language", this.inputLanguage);
+        obj.put("encoding", this.inputEncoding);
+        return obj;
+    }
+
+    @Override
+    public void setFilter(JSONObject filter) {
+        Iterator<String>  keys = filter.keySet().iterator();
+        while(keys.hasNext()){
+            String key = keys.next();
+            switch(key){
+                case Constants.configEntityTypeFilter : {
+                    this.typeFilter = (List<String>) filter.get(key);
+                    break;
+                }
+                case Constants.configEntitySalienceFilter : {
+                    this.salienceFilter = (Double) filter.get(key);
+                    break;
+                }
+                default :{
+                    //TODO: Error
+                }
+            }
+        }
+    }
+
+    @Override
+    public JSONObject getFilter() {
+
+        JSONObject obj = new JSONObject();
+        obj.put("type", this.typeFilter);
+        obj.put("salience", this.salienceFilter);
+        return obj;
+    }
+
+    private Boolean isSatisfyFilter(Entity.Type type, float salience){
```

Comment:
1. Rename to either isFilterSatisfied or fiterEntity.
2. float salience --> double salience
3. Boolean --> boolean